cmd/ipfs-cluster-ctl/formatters.go

```diff
@@ -223,3 +224,9 @@ func trackerStatusAllString() string {
 	sort.Strings(strs)
 	return strings.Join(strs, "\n")
 }
+
+func textFormatPrintList(obj []string) {
```

Comment:
add a 

```
case string:
   fmt.Println(resp)
```

towards the top of `textFormatObject`

the `[]string` case should do the loop of the items like the other slice types (for consistency)